airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -334,6 +335,8 @@ def __init__(
         self.env_vars = env_vars
         if pod_runtime_info_envs:
             self.env_vars.extend([convert_pod_runtime_info_env(p) for p in pod_runtime_info_envs])
+        mask_secret(self.env_vars)
+        self.env_vars = redact(self.env_vars)
```

Comment:
can we test it to make sure values are masked?

Reply:
Yes, I have added the testing results in the PR description. @eladkal

Reply:
Unit test cases are needed for this change.

Reply:
Yeah, looking at it.